pylint/checkers/classes/class_checker.py

```diff
@@ -428,7 +429,9 @@ def _has_same_layout_slots(slots, assigned_value):
     return False
 
 
-MSGS = {  # pylint: disable=consider-using-namedtuple-or-dataclass
+MSGS: dict[
```

Comment:
I'd rather put that directly in the checker as ``msgs`` so the typing is implicit, what do you think ?

Reply:
That doesn't work as dictionaries are invariant. See https://mypy.readthedocs.io/en/stable/common_issues.html#variance